include/fc/log/logger.hpp

```diff
@@ -77,31 +87,31 @@ namespace fc
 #define fc_dlog( LOGGER, FORMAT, ... ) \
   FC_MULTILINE_MACRO_BEGIN \
    if( (LOGGER).is_enabled( fc::log_level::debug ) ) \
-      (LOGGER).log( FC_LOG_MESSAGE( debug, FORMAT, __VA_ARGS__ ) ); \
+      SPDLOG_LOGGER_DEBUG((LOGGER).get_agent_logger(), FC_FMT( FORMAT, __VA_ARGS__ )); \
```

Comment:
Is there a reason to use `fc::logger`'s dictionary of loggers as opposed to `spdlog`'s dictionary of loggers?
https://github.com/gabime/spdlog#create-stdoutstderr-logger-object

Reply:
The main reason of not directly using `spdlog`'s dictionary of loggers is that `fc` logging level config code and variant `fc` logger definition code could be re-used by defining `spdlog`'s loggers as agent loggers of `fc`.